README.md

```diff
@@ -283,6 +283,11 @@ HTML::Restrict recognizes:
     enclosed content you'd like to remove.  This feature strips script and style
     tags by default.
 
+- `process_text => [0|1]`
+
+    By default all text will be processed.  Set this value to 0 in order to disable this behaviour.  Please be advised this is a security risk. Use caution when disabling this parameter.
+
+
```

Comment:
These docs are in the wrong location. The `README.md` file is a generated file, based on the docs in the `lib/HTML/Restrict.pm` file.

I also don't find them very clear. What does "processed" mean in this context?

Reply:
Maybe `filter_text()` would be clearer?

Reply:
@haarg 
I didnt realize they were generated from module I can make that change easily enough.

Reply:
Let me know how you want this to be named. Variable naming is usually the hardest part of coding!

We have a release starting today so I will try to get these in asap

Reply:
I think `filter_text()` would work. If there are little things like naming, I can just update the commits before merge. That's no problem at all. Thanks for all of your help with this.